instruments/toptica/topmode.py

```diff
@@ -310,7 +342,15 @@ def reference(self, param):
         :return: Response to the reference request
         :rtype: `str`
         """
-        return self.query("(param-ref '{})".format(param))
+        # the toptica topmode termination character is \r\n,
+        # but unfortunately serial communicator only allows for single
+        # character terminators.
+        # toptica also returns all strings encapsulated by double quotations.
+        response = self.query("(param-ref '{})".format(param))
+        response = sub("\r", "", response)
```

Comment:
For generic substitutions (without specifying other regex stuff, like beginning/end of string) you can always use the Python `String.replace` method, but I'm cool with this.